fuse.js

```diff
@@ -8,13 +8,62 @@
             tabs: 'fuseSettingsPanel__tabs',
             borisChen: 'fuseSettingsPanel__tabs__borisChen',
             subvertADown: 'fuseSettingsPanel__tabs__subvertADown',
-            csv: 'fuseSettingsPanel__tabs__csv'
+            customData: 'fuseSettingsPanel__tabs__customData'
         },
         localStorage: 'fuseStorage'
     }
 
     runAutoUpdate();
 
+    const DSTNames = getDSTNames();
+
+    function getDSTNames() {
```

Comment:
Moved this to the root of the script so that it gets calculated once since this now outputs a dictionary with 192 DST name permutations